src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -280,20 +283,62 @@ protected virtual IObjectGraphType ToObjectGraphType(GraphQLObjectTypeDefinition
             return type;
         }
 
+        private void InitializeField(FieldConfig config, Type parentType)
+        {
+            config.ResolverAccessor ??= parentType.ToAccessor(config.Name, ResolverType.Resolver);
+
+            if (config.ResolverAccessor != null)
+            {
+                config.Resolver = new AccessorFieldResolver(config.ResolverAccessor, ServiceProvider);
+                var attrs = config.ResolverAccessor.GetAttributes<GraphQLAttribute>();
+                if (attrs != null)
+                {
+                    foreach (var a in attrs)
+                        a.Modify(config);
+                }
+            }
+        }
+
+        private void InitializeSubscriptionField(FieldConfig config, Type parentType)
```

Comment:
```suggestion
        protected virtual void InitializeSubscriptionField(FieldConfig config, Type parentType)
```

Reply:
Both methods are called from other virtual methods. I do not consider it right to make them virtual. In fact, it is just a piece of private implementation. If the user wants to change the implementation, it will override the calling virtual method.

Reply:
Ok.